src/openassetio-python/package/openassetio/managerApi/ManagerInterface.py

```diff
@@ -513,6 +520,24 @@ def getWithRelationship(
         logging and provides access to the openassetio.managerApi.Host
         object representing the process that initiated the API session.
 
+        @param successCallback Callback that must be called for each
+        successful relationship query for an input entity reference. It
+        should be given the corresponding index of the entity reference
+        in `entityReferences` along with a list of entity references for
+        entities that are related by the input entity by the specified
+        relationship. If there are no relations, an empty list should be
+        used. The callback must be called on the same thread that
```

Comment:
[Consider]
Similarly ambiguous to the docs in manager. Perhaps?

```suggestion
        relationship. If there are no relations, an empty list should be
        provided to the callback. The callback must be called on the same thread that
```